.env-example

```diff
@@ -0,0 +1,52 @@
+# domain name
+# set to localhost if developing locally
+DOMAIN=perf.test.fedcloud.edu
+
+# NGINX update credentials
+NGINX_API_USER=user
+NGINX_API_PASS=pass
```

Comment:
Are this credentials safe?
Shall we move them to secrets?

Reply:
These are just basic credentials for the certbot https cert update, they do not leave the container and are not accessible anyway. In theory, there doesn't _need_ to be a config setting for this, but I left it in because the configuration I based it on did.

Reply:
Update from meeting: this endpoint may actually be exposed, so it should be configurable in a secret instead.